runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -334,6 +343,29 @@ public void onWidgetLoaded(WidgetLoaded event)
 			return;
 		}
 
+		if (!(event.getGroupId() == WidgetID.CLUE_SCROLL_REWARD_GROUP_ID) && config.chestLootChat())
```

Comment:
Is this here because the game already shows you clue scroll price? This should be written as `event.getGroupid() != WidgetID.CLUE_SCROLL_REWARD_GROUP_ID`